Sources/Plasma/PubUtilLib/plCompression/plZlibCompress.cpp

```diff
@@ -138,7 +141,7 @@ bool  plZlibCompress::UncompressFile( const char *compressedPath, const char *de
         {
             for( ;; )
             {
-                length = gzread( inFile, buffer, sizeof( buffer ) );
+                uint32_t length = gzread( inFile, buffer.get(), kGzBufferSize );
```

Comment:
`gzread` returns an `int` (which makes the next line, `if( length < 0 )` make more sense)